readme.md

```diff
@@ -49,7 +49,8 @@ store.uploadExisting(myZipFile, token).then(res => {
 ```javascript
 const target = 'default'; // optional. Can also be 'trustedTesters'
 const token = 'xxxx'; // optional. One will be fetched if not provided
-store.publish(target, token).then(res => {
+const deployPercentage = '25'; // optional. Will default to 100%.
```

Comment:
This package should accept a number, not a string

Reply:
it's updated